rnwinrt/rnwinrt/react/ReactFileGenerator.cpp

```diff
@@ -410,9 +410,24 @@ namespace rnwinrt::enums::%
         if (!field.Constant()) // Enums have a 'value__' data member we need to ignore
             continue;
 
+        auto constant = field.Constant();
+        std::string_view fieldValueAsString;
```

Comment:
```suggestion
        std::string fieldValueAsString;
```
This needs to be a `std::string` otherwise the value will be destructed & its memory will be reclaimed before this is used

Reply:
I followed the pattern that is already used in this file. For example, see `kindNs` in `write_rnwinrt_namespace_cpp_files`.

Reply:
That's because `kindNs` is being assigned a string literal, not a `std::string`

Reply:
Got it